packages/main/src/PopoverRegistry.js

```diff
@@ -0,0 +1,112 @@
+import { isClickInRect } from "./PopupUtils.js";
+
+import { getOpenedPopups, addOpenedPopup, removeOpenedPopup } from "./OpenedPopupsRegistry.js";
+
+const intervalTimeout = 300;
+const updateInterval = null;
+const openedRegistry = [];
+
+const repositionPopovers = event => {
+	openedRegistry.forEach(popover => {
+		popover.reposition();
+	});
+};
+
+const attachGlobalScrollHandler = () => {
+	document.body.addEventListener("scroll", repositionPopovers, true);
+};
+
+const detachGlobalScrollHandler = () => {
+	document.body.removeEventListener("scroll", repositionPopovers, true);
+};
+
+const runUpdateInterval = () => {
+	return setInterval(() => {
```

Comment:
IMO it would be clearer to assign to updateInterval here, this way it will harmonize with stopUpdateInterval, which works with the variable directly